packages/components/src/components/Ansi/Ansi.js

```diff
@@ -0,0 +1,268 @@
+/*
+Copyright 2019-2020 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import React from 'react';
+import PropTypes from 'prop-types';
+import { colors, textStyles } from './defaults';
+
+const linkifyIt = require('linkify-it')().tlds(require('tlds'));
+
+const Ansi = ({ children }) => {
+  let properties = {
+    foregroundColor: null,
+    backgroundColor: null,
+    bold: false,
+    italic: false,
+    underline: false,
+    conceal: false,
+    cross: false
+  };
+
+  let styles = {};
+  let text = '';
+  let line = [];
+
+  const reset = () => {
+    properties = {
+      foregroundColor: null,
+      backgroundColor: null,
+      bold: false,
+      italic: false,
+      underline: false,
+      conceal: false,
+      cross: false
+    };
+  };
+
+  const enable = flag => {
+    properties[flag] = true;
+  };
+
+  const disable = flag => {
+    properties[flag] = false;
+  };
+
+  const getXtermColor = commandStack => {
+    if (commandStack.length >= 2 && commandStack[0] === '5') {
+      commandStack.shift();
+      const colorIndex = +commandStack.shift();
+      if (colorIndex >= 0 && colorIndex <= 255) {
+        return colors[colorIndex];
+      }
+    }
+    return null;
+  };
+
+  const getTermColor = (colorIndex, type) => {
+    if (colorIndex > 7) {
+      return null;
+    }
+    if (type === 'l') {
```

Comment:
what's `l` here? From context I'm guessing `light` or similar (referencing the set of alternative 'bright' colours?) It would be good to use a descriptive name to make this clear.

Reply:
Renamed to use 'light'